### ISSUE TITLE
Allow tooltip to be opened to a standalone window.

### ISSUE THREAD

**Issue by chancedj** [chancedj/LockedOut#83]

**Comment by Trid3nt1** [comment]
I was about to make this suggestion but it seems it may be due? That's fantastic.. Really, a huge QoL improvement.

**Comment by chancedj** [comment]
yes, i just need to understand the wow forms a little better before i do this.  i had started on it but it became too diffucult, so i ditched the branch and will start it again here soon.  possibly part of this release, but likely 1.9.
